silverstripe/framework/CVE-2023-22729.yaml

```diff
@@ -0,0 +1,11 @@
+title:     "CVE-2023-22729 - Open redirect vulnerability on CMSSecurity relogin screen"
+link:      https://www.silverstripe.org/download/security-releases/cve-2023-22729
+cve:       CVE-2023-22729
+branches:
+    4.0.x:
+        time:     2023-04-26 00:44:05
+        versions: ['>=4.0.0', '<4.12.5']
```

Comment:
Those entries for 4.0.x are useless. The range is already covered by you 4.12.x range (in all of them)

Reply:
Fair enough, though in that case the readme should probably be updated to clarify what the "branches" information is for. My understanding from what is written is that each affected branch should have its own entry. 

Regardless, I'll update the pr to remove the 4.0.x branches tomorrow.

Reply:
We need multiple branches only when the issue gets patched in multiple branches and so a single range is not enough. Auditing tools will compare the version you use to those ranges.

Reply:
Fair enough. I think the readme should be more explicit about that, but that's another discussion. I'll make the change to orrow as mentioned earlier.

Reply:
don't hesitate to open a PR improving the readme :)

Reply:
I honestly don't feel like I understand the intention of the "branches" keys well enough to update the readme 😅 I'll leave that to someone who knows this project better.

Reply:
`4.0.x` branches removed